libr/util/str_trim.c

```diff
@@ -82,6 +82,10 @@ R_API char *r_str_trim_dup(const char *str) {
 
 R_API void r_str_trim(char *str) {
 	r_return_if_fail (str);
+#if 1
+	r_str_trim_head (str);
```

Comment:
why calling this if in the next line you call r_str_trim_head_tail, which does both head and tail?

Reply:
because thats what r_str_trim() do. ts an alias for head_tail

Reply:
If it is an alias, just make it a define in the r_str.h header  or implement it as:
```C
R_API void r_str_trim(char *str) {
   r_str_trim_head_tail (str);
}
```
don't duplicate the implementation in both functions.

Reply:
nvm i just removed the dupe